Specs/NumberWithUnit/Portuguese/CurrencyModel.json

```diff
@@ -1584,7 +1584,7 @@
   },
   {
     "Input": "custa apenas 15 dólares 50.",
-    "NotSupported": "javascript, java",
+    "NotSupported": "javascript, java, python",
```

Comment:
Why is this now broken in Python?

Reply:
Because we added "dólares" to the prefix list in order to extract patterns like "dólarer 22,5 milhões" and then modified the method SelectCandidates in .NET to correctly handle cases like "15 dólares 50", "3 $50"... 
But the method is missing in Python, so now it returns "dólares 50" instead  of "15 dólares 50".

Reply:
I could add SelectCandidates in Python if necessary.

Reply:
If that doesn't take you much time and block other work, please do. Thanks.

Reply:
done